tests/e2e/smoke.json

```diff
@@ -0,0 +1,65 @@
+{
+	"info": {
+		"_postman_id": "138a4806-74f8-406a-90cb-c2acb50630ea",
+		"name": "Personal Demographics Service - SmokeTest",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Retrieve Patient",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"id": "9ba9f64b-ae66-4e9f-93e8-e74f28f48a5c",
+						"exec": [
+							"pm.test(\"Returns Status code 200\", function () {",
+							"    pm.response.to.have.status(200);",
+							"});",
+							"",
+							"pm.test(\"Returns Patient\", function () {",
+							"    var jsonData = pm.response.json();",
+							"    console.log(\"Response: \" + JSON.stringify(jsonData, null, 2));",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "GET",
+				"header": [
+					{
+						"key": "NHSD-Identity-UUID",
+						"value": "1234567890",
+						"disabled": true
+					},
+					{
+						"key": "Authorization",
+						"value": "Bearer  {{token}}"
+					},
+					{
+						"key": "NHSD-Identity-JWT",
+						"value": "{{jwt}}"
+					},
+					{
+						"key": "NHSD-Session-URID",
+						"value": "1234567890"
+					}
+				],
+				"url": {
+					"raw": "{{environment}}/Patient/5900018512",
```

Comment:
Why doesn't this patient start with a 9?

Reply:
@gjugpal

Reply:
This is just a random Patient that I created on VEIT07. I'll be adding more tests to this collection in due course.

Reply:
@gjugpal  the convention is that test NHS IDs all start with a 9. One that doesn't makes us nervous!

Reply:
@alexhawdon I wasn't aware of that, the DSA tool which we use to create/amend patients, auto generates NHS numbers on create. To confirm this is a Patients NHS number not a clinicians NHS Id used to login to the client.

Reply:
The only spine env this regression pack will run against is VEIT07 so we should be safe

Reply:
Ideally if we could hit a healthcheck endpoint that would be better

Reply:
I've repointed it to `/healthcheck`